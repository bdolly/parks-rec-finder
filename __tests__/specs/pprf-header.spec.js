import { mount } from 'vue-test-utils'
import { createRenderer } from 'vue-server-renderer'

import pprfHeader from '@/components/pprf-header.vue'

describe('pprf-header.vue', () => {
  const wrapper = mount(pprfHeader)
  it('sanity check', () => {
    expect(true).toBe(true)
  })

  it('matches snapshot', () => {
    const renderer = createRenderer()
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })

})
