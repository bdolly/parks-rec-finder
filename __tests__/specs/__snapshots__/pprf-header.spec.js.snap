// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pprf-header.vue matches snapshot 1`] = `
<header class="pprf-header">
    <!---->
</header>
`;
